pkg/volume/host_path/host_path.go

```diff
@@ -47,11 +47,8 @@ func (plugin *hostPathPlugin) Name() string {
 	return hostPathPluginName
 }
 
-func (plugin *hostPathPlugin) CanSupport(spec *api.Volume) bool {
-	if spec.HostPath != nil {
-		return true
-	}
-	return false
+func (plugin *hostPathPlugin) CanSupport(spec *volume.InternalVolume) bool {
+	return spec.VolumeSource.HostPath != nil || spec.PersistentVolumeSource.HostPath != nil
```

Comment:
It's what I use for development and testing.  The API comments explicitly flag it as dev/test use only.